dtmi/murata/demo_iot-1.json

```diff
@@ -0,0 +1,42 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:murata:demo_iot;1",
+    "@type": "Interface",
+    "displayName": "Murata Sensors Demo Model on B-U585I-IOT02A",
+    "description": "Murata Sensors Demo Model lists interfaces, telemetry, and command options",
+    "contents": [
+        {
+            "@id": "dtmi:murata:b_u585i_iot02a:standard_fw:std_comp;3",
```

Comment:
I dont think you need this @id, but if you do it should follow the root id.

https://github.com/Azure/iot-plugandplay-models-tools/wiki/Resolution-Convention#unique-id-elements